scheduler/src/cook/mesos/share.clj

```diff
@@ -60,14 +60,14 @@
    \"default\" user. If there is NO \"default\" value for a specific type,
    return Double.MAX_VALUE."
   [db user]
-  (->> (util/get-all-resource-types db)
+  (->> [:cpus :mem]
```

Comment:
Which tests broke? I would prefer not to have to do this but on the other hand, don't want you going down a rabbit hole